Source/Core/Enum/Encounter.hpp

```diff
@@ -35,8 +35,9 @@ enum class Encounter : u8
     SuperRod,
     SpecialSuperRod,
     Static,
+    SafariZone,
```

Comment:
Why was safarizone added back? This is just deadcode right now

Reply:
Removed